substrate/frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -873,6 +909,9 @@ pub mod pallet {
 			// `SignedMaxSubmissions` is a red flag that the developer does not understand how to
 			// configure this pallet.
 			assert!(T::SignedMaxSubmissions::get() >= T::SignedMaxRefunds::get());
+
+			// idem.
+			assert!(T::SignedMaxSubmissions::get() >= T::SignedWhitelistMax::get());
```

Comment:
Wondering if we require `SignedWhitelistMax` as a perbill of `SignedMaxSubmissions`, would it be better? One less thing a new dev could do wrong with the configuration?